tests/foundry/client-esm/data/calendar.mjs.test-d.ts

```diff
@@ -1,54 +1,63 @@
 // Copied implementation from Foundry's calendar.mjs
-const _gregorian: foundry.data.CalendarData.Config = {
+const _gregorian: foundry.data.CalendarData.CreateData = {
```

Comment:
It might be worth switching to satisfies. `const x: T = ...` can have some strange side effects.